unit/unittype.go

```diff
@@ -212,8 +214,10 @@ func SymbolExists(symbol string) bool {
 // mass to get a force. See the package documentation for further explanation.
 type Unit struct {
 	dimensions Dimensions
-	formatted  string
 	value      float64
+
+	mu        sync.RWMutex
```

Comment:
We should just ditch this and recompute the formatting string on the fly. It's not that expensive an operation, it's way simpler to avoid the mutex, and it simplifies other methods because we don't track if we've already computed it or not.

Reply:
I'm not convinced, but we can add it back if needed.

OK